packages/yew/Makefile.toml

```diff
@@ -10,14 +10,15 @@ args = [
     "--headless",
     "--",
     "--features",
-    "wasm_test"
+    "csr,hydration,ssr"
```

Comment:
`--all-features` now that csr, ssr, and hydration are only features left

Reply:
Ah I think in this case, the optional `tokio` dependency should be disabled, still. I was a bit hasty. Reverting to "csr,hydration,ssr".